build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -336,7 +336,7 @@ def populateSettingValuesFromConfigMap(parsedConfig)
           puts "Using config map value: AZMON_FBIT_MEM_BUF_LIMIT = #{@promFbitMemBufLimit.to_s + "m"}"
         end
       end
-      proxy_config = parseConfigMap[:agent_settings][:proxy_config]
+      proxy_config = parsedConfigMap[:agent_settings][:proxy_config]
```

Comment:
so this never worked? I wonder why no customer faced an issue with this

Reply:
> so this never worked? I wonder why no customer faced an issue with this

this is not bug. parseConfigMap invokes the function which returns same result.